apps/dashr-svm/app.R

```diff
@@ -0,0 +1,514 @@
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+
```

Comment:
Might not be a bad idea to add a similar explanatory header to the R app here as found in the Python one:

https://github.com/plotly/dash-svm/blob/b2c518156c29afa3f719af87d8c7c5d74a993e73/app.py#L1-L8